webdriver-spec.html

```diff
@@ -3946,6 +3946,9 @@
  <li><p>Let <var>centre point</var> be a pair of <var>rect</var>’s
   (<a>rectangle x coordinate</a> + <var>horizontal centre offset</var>)
   and (<a>rectangle y coordinate</a> + <var>vertical centre offset</var>)
+  
+ <li><p>If <a>command</a> is intercepted by an <var>element</var> with a higher 
+  z-index, an <a>element intercepting click</a> <a>error</a> is returned.
```

Comment:
We might want to reference elementsFromPoint. Where this exception would get thrown if the originating element is not in the set returned from that function provided the coordinates just calculated. 

http://www.w3.org/TR/cssom-view-1/#extensions-to-the-document-interface